src/cljs/zetawar/data.cljs

```diff
@@ -31,6 +31,7 @@
     :build-unit-button "Build"
     :attack-unit-button "Attack"
     :repair-unit-button "Repair"
+    :repair-other-button "Repair Other"
```

Comment:
Although we need to distinguish between self repair and repairing other units at the action level, I think it's ok for the buttons to have the same label. The context makes the type of repair clear.